### PR TITLE
vmui/logs: add bar chart

### PR DESCRIPTION
**Pull request by Loori-R**
### Describe Your Changes

- Added a bar chart displaying the number of log entries over a time range.
   #6404
- When `_msg` is empty, all fields are displayed in a single line.
- Added double quotes when copying pairs: `key: "value"`.
- Minor style adjustments.

image
image

### Checklist

The following checks are **mandatory**:

- [ ] My change adheres [VictoriaMetrics contributing guidelines](https://docs.victoriametrics.com/contributing/).

### CODE REVIEW

**Comment by hagen1778** [comment]
@Loori-R is this ready for review or there is some additional work to do?

**Comment by Loori-R** [comment]
@hagen1778, I have just updated the code, and it is now ready for review.

File: app/vmui/packages/vmui/src/pages/ExploreLogs/ExploreLogsBarChart/ExploreLogsBarChart.tsx [hunk 1]
```diff
@@ -31,7 +31,18 @@ const ExploreLogsBarChart: FC<Props> = ({ logHits, error, loaded }) => {
...
   }, [logHits]);
 
-  const noData = data.some(d => d.length === 0);
+  const noDataMessage: string = useMemo(() => {
+    const noData = data.every(d => d.length === 0);
+    const noTimestamps = data[0].length === 0;
+    const noValues = data[1].length === 0;
+    if (noData) {
+      return "No logs volume available\nNo volume information available for the current queries and time range.";
+    } else if (noTimestamps) {
```
Comment (dmitryk-dk) [inline app/vmui/packages/vmui/src/pages/ExploreLogs/ExploreLogsBarChart/ExploreLogsBarChart.tsx:40]: nit: else is redundant

**Review by dmitryk-dk — APPROVED** [review]
Overall looks good!

File: app/vmui/packages/vmui/src/pages/ExploreLogs/ExploreLogsBarChart/ExploreLogsBarChart.tsx (hunk 1 above)
Comment (Loori-R) [reply app/vmui/packages/vmui/src/pages/ExploreLogs/ExploreLogsBarChart/ExploreLogsBarChart.tsx:40]: In cases of aggregation, the API may return a value without timestamps; in this case, the message will explicitly indicate this.

**Review by hagen1778 — APPROVED** [review]
LGTM

**Pull request merged**